lib/components/user/places/place-editor.js

```diff
@@ -113,11 +113,14 @@ class PlaceEditor extends Component {
             <PlaceLocationField
               className='form-control'
               inputPlaceholder={isFixed ? `Search for ${place.name} location` : 'Search for location'}
+              // Add a flag for mobile views to adjust dropdown menu position/width.
+              // The component's 'static' prop is not suitable for mobile views here, as it would
+              // show the list of places over the Back/Save buttons, making these buttons unreachable.
+              isMobile={isMobile()}
               location={makeLocationFieldLocation(place)}
               locationType='to'
               onLocationSelected={this._handleLocationChange}
               showClearButton={false}
-              static={isMobile()}
```

Comment:
Sigh... I forgot that showing the field in mobile view is why we opted for the buttons above instead of below. We may want to change this so that in mobile mode the buttons render at the top of the screen and in desktop they're below the location field.

Reply:
I'll add that. The other thing we could do in mobile (in another PR) is to flush the back/save buttons to the bottom of the screen so that they are always shown, but that may be also be tricky...

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/326199f4e5bdd0e8a67538c01021973b558fc316.